src/lib/auth.test.ts

```diff
@@ -371,6 +374,20 @@ describe("teacher authentication", () => {
     })
     .get("")
     .reply(200, PARTIAL_RAW_OFFERING_INFO);
+
+    /*
+    FIXME TODO:  I can't get nock to match this request:
+        Nock: No match for request {
+      "method": "GET",
+      "url": "https://learn.staging.concord.org/api/v1/offerings/?user_id=217"
+    }
+    One thing to do is to remove fetching portal data (classes etc) from the
+    auth.ts method.  Move that out into app.ts so we can test in isolation.
```

Comment:
Is this comment still relevant now that portal code has moved to `portal-api.ts`?